Language/Libraries_and_Scripts/definition_syntax_t18.dart

```diff
@@ -19,8 +19,9 @@
 /// library begins with the word library (possibly prefaced with any applicable
 /// metadata annotations), followed by a qualified identifier that gives the
 /// name of the library.
```

Comment:
I think some of this is now out of date, corresponding to the change to allow "library;" with no name?

Reply:
I'm wondering: Do we gain anything by having the `library;` directive in Language/Libraries_and_Scripts/definition_syntax_t18_lib.dart and importing that library into this one? It seems likely to me that we could just avoid having any connection to multiple libraries and reporting errors (or not, as it would be expected today) in a different library than the entry point.

We would then add `library;` to this library (expecting no errors), remove the import, delete Language/Libraries_and_Scripts/definition_syntax_t18_lib.dart, and thus get to a state where this library is a non-problem relative to the test runner's treatment of errors outside the entry point.

Reply:
Looks good now! Should perhaps change the description as mentioned below.